### PR TITLE
Make README the default problem view.

### PR DESCRIPTION
**Pull request by k3rni**
Addresses #3025 in the laziest way possible.

### CODE REVIEW

**Comment by kytrinyx** [comment]
We need to make sure that we don't create a 404 if someone has linked to this from a blog post or tweet, or if we're linking to them from various bits of documentation in the language tracks.

I think it would make sense to have both URLs be specific URLs:

- `GET /exercises/:track_id/:slug/test-suite`
- `GET /exercises/:track_id/:slug/readme`

and then redirect the `GET /exercises/:track_id/:slug` endpoint to the one that we want.

We also have a couple of other links to those pages in the site itself. Check out the results from this:

```
$ git grep href.*exercises.*slug app/views/
```

**Comment by Insti** [comment]
I like the `UserExercises` extraction.

**Comment by Insti** [comment]
It would be good to add some test coverage for `/app/routes/exercises.rb`

File: test/app/exercises_test.rb [hunk 1]
```diff
@@ -58,4 +58,53 @@ def test_remove_request_for_help
...
+    end
+  end
+
+  def test_exercise_api_error
+    X::Xapi.stub :get, [500, {error: 'Whoops'}.to_json] do
+      get "/exercises/foo/bar/readme"
+
+      assert_equal 302, last_response.status
+      assert_equal 'http://example.org/', last_response.location
+      # require 'byebug'; byebug
```
Comment (kytrinyx) [inline test/app/exercises_test.rb]: Would you remove the commented out code, please?

**Comment by kytrinyx** [comment]
Merged in 5ecb3369be32953dc1a65c668be3dff520e4d57f

**Pull request closed without merging**